docs/_docs/0_4_docker_demo.cn.md

```diff
@@ -1106,6 +1106,9 @@ and compose scripts are carefully implemented so that they serve dual-purpose
 2. For running integration-tests, we need the jars generated locally to be used for running services within docker. The
    docker-compose scripts (see `docker/compose/docker-compose_hadoop284_hive233_spark231.yml`) ensures local jars override
    inbuilt jars by mounting local HUDI workspace over the docker location
+3. Due to the docker images have mounted local HUDI workspace, so any changes happen in the workspace would reflect into the 
```

Comment:
Reword To :  "As these docker containers have mounted local HUDI workspace, any changes that happen in the workspace would automatically reflect in the  containers. This is a convenient way for developing and verifying Hudi for 
 developers who do not own a distributed environment. Note that this is how integration tests are run."